packages/datagateway-dataview/src/page/pageContainer.component.tsx

```diff
@@ -477,47 +478,56 @@ class PageContainer extends React.Component<
   public render(): React.ReactElement {
     return (
       <Paper square elevation={0} style={{ backgroundColor: 'inherit' }}>
-        <NavBar
-          entityCount={this.props.entityCount}
-          cartItems={this.props.cartItems}
-          navigateToSearch={this.props.navigateToSearch}
-          navigateToDownload={this.props.navigateToDownload}
-        />
+        {this.props.location.pathname === paths.homepage ? (
+          <PageRouting
+            view={this.props.savedView}
+            location={this.props.location}
+          />
+        ) : (
+          <>
+            <NavBar
+              entityCount={this.props.entityCount}
+              cartItems={this.props.cartItems}
+              navigateToSearch={this.props.navigateToSearch}
+              navigateToDownload={this.props.navigateToDownload}
+            />
```

Comment:
Done like this to ensure the nav bar and a big footer don't render on the homepage (along with other unnecessary pageContainer components). Otherwise, we get something like in the image. Still not overly sure of this code structure. Thoughts, @louise-davies ?
![Capture](https://user-images.githubusercontent.com/71134574/118178364-c82e4100-b42b-11eb-8e2f-4bd093a6c381.PNG)

Reply:
It might make more sense to have the `HomePage` component back in the `PageContainer` component similar to the original logic and instead just add a `Switch` component rather than an `if` statement and two routes - one for the `HomePage` which has no `NavBar` and one catchall route for the layout stuff for the `NavBar` and thus the `PageRouting` component. Again, I would just say we would need to memoize the layout stuff + NavBar to optimise. Let me know if you'd like a pair programming session to explain this a little better @sam-glendenning